src/components/QueryBuilder.vue

```diff
@@ -51,6 +53,7 @@ import StepFormsComponents from './stepforms';
 })
 export default class QueryBuilder extends Vue {
   version = version; // display the current version of the package
+  editedStepBackendError: string | undefined = undefined;
```

Comment:
Might not be the better implementation but we need to keep data even if the store is refreshed (data is updated when entering the step so the error won't exist anymore)

Reply:
It's a good idea to have done this! With a quick comment describing the issue it'll perfectly do the job :+1:

Reply:
I'll do it :ok_hand: